python/build.py

```diff
@@ -149,16 +155,19 @@ def main():
     if mode == 'upload' and env_pypi_pwd == '':
         raise RuntimeError("Missing environment variable PYPI_PWD")
 
+    if mode == 'upload' and args.testpypi:
+        pypi_user = '__token__'
+        test_pypi = '--repository testpypi '
+
     if not args.skip_build:
         build()
 
     if mode == 'build':
         return
     elif mode == 'upload':
-        os.system(
-            '{} -m twine upload dist/* --verbose -u yuanming-hu -p {}'.format(
-                get_python_executable(),
-                '%PYPI_PWD%' if get_os_name() == 'win' else '$PYPI_PWD'))
+        os.system('{} -m twine upload {}dist/* --verbose -u {} -p {}'.format(
```

Comment:
nit: Just create different command lines depending on whether `args.testpypi` if True. `test_pypi` itself isn't a reasonable name in every case :-)